client/packages/invoices/src/OutboundShipment/DetailView/OutboundLineEdit/Allocation.tsx

```diff
@@ -78,16 +104,9 @@ const AllocationInner = ({ item }: { item: DraftItem }) => {
               justifyContent: 'center',
             }}
           >
-            {t('label.available-quantity', {
-              number: sumAvailableQuantity(draftLines).toFixed(0),
-            })}
+            {getAvailableQuantity()}
           </Typography>
         </Grid>
-
-        <Grid style={{ display: 'flex' }} justifyContent="flex-end" flex={1}>
-          <ModalLabel label={t('label.unit')} justifyContent="flex-end" />
-          <BasicTextInput disabled sx={{ width: 150 }} value={item?.unitName} />
```

Comment:
Removed now that unit name is included in the `Available quantity: 7 vials` message